src/modules/DressMe/Sizes.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { Field, Formik, useFormik } from "formik";
+import { validate } from "./validate";
+
+const Sizes = () => {
+  return (
+    <Container>
+      <Select name="shirtsize" style={{ display: "block" }}>
+        <option value="" label="Pick your shirt size">
+          Select a color
+        </option>
+        <option value="xs" label="xs">
+          XS
+        </option>
+        <option value="s" label="s">
+          S
+        </option>
+
+        <option value="m" label="m">
+          M
+        </option>
+        <option value="l" label="l">
+          L
+        </option>
+        <option value="xl" label="xl">
+          XL
```

Comment:
you can construct these using a dictionary (object) for the different values and map them to <option/>
the options data will look like: [{value: "xs", label: "XS"}]
and then you can map them to <option> components and pass the relevant props